code/game/objects/items/draftingchalk.dm

```diff
@@ -18,37 +18,77 @@
 			smooth_icon(C)
 
 /obj/item/pen/drafting
-	name = "white drafting chalk"
-	desc = "A piece of white chalk for marking areas of floor."
+	name = "white chalk"
+	desc = "A piece of white chalk for marking areas of floor, or for drawing."
 	icon = 'icons/obj/crayons.dmi'
 	icon_state = "dchalk"
 	color = "#FFFFFF"
 	var/colorName = "whitec"
+	var/linemode = TRUE
+	var/instant = 0
 
 /obj/item/pen/drafting/attack_self(var/mob/user)
+	if(linemode)
+		linemode = FALSE
+		to_chat(user, SPAN_SUBTLE("You will now draw freestyle."))
+	else
+		linemode = TRUE
+		to_chat(user, SPAN_SUBTLE("You will now draw straight lines."))
 	return
 
-/obj/item/pen/drafting/afterattack(turf/target, mob/user, proximity)
-	if (!proximity || !istype(target, /turf/simulated) || target.density || target.is_hole)
-		return
-
-	to_chat(user, "You start marking a line on [target].")
+/obj/item/pen/drafting/examine(mob/user)
+	. = ..()
+	to_chat(user, SPAN_SUBTLE("Use the chalk on itself to change what you will draw."))
 
-	if (!do_after(user, 1 SECONDS, act_target = target))
+/obj/item/pen/drafting/afterattack(turf/target, mob/user, proximity)
+	if(linemode)
+		if (!proximity || !istype(target, /turf/simulated) || target.density || target.is_hole)
+			return
+
+		to_chat(user, "You start marking a line on [target].")
+
+		if(!do_after(user, 1 SECONDS, act_target = target))
+			return
+
+		for (var/obj/effect/decal/cleanable/draftingchalk/C in target)
+			qdel(C)
+
+		to_chat(user, "You mark a line on [target].")
+
+		var/obj/effect/decal/cleanable/draftingchalk/C = new(target)
+		C.color = color
+		target.add_fingerprint(user)
+	else	//code copied graciously from crayon.dm
+		if(!proximity) 
+			return 
+		if(istype(target,/turf/simulated/floor))
+			var/originaloc = user.loc
+			var/drawtype = input("Choose what you'd like to draw.", "Chalk freestyle") in list("graffiti","rune","letter","arrow")
+			if (user.loc != originaloc)
+				to_chat(user, "<span class='notice'>You moved!</span>")
+				return
+
+			switch(drawtype)
+				if("letter")
+					drawtype = input("Choose the letter.", "Chalk freestyle") in list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+					to_chat(user, "You start drawing a letter on the [target.name].")
```

Comment:
You don't have to copypaste all these lines, just set a variable then at the end after the switch do:
`to_chat(user, SPAN_NOTICE("You start drawing a [draw_thing] on [target]."))`
Similiarly, don't do `the [target.name]`, just do `on [target]` as it applies "the" as needed

Reply:
Done.  Also crayon copypasta, but fixed!